src/api/app/models/bs_request_action.rb

```diff
@@ -822,6 +822,11 @@ def uniq_key
     ' '
   end
 
+  # This is duplication of part of the code in the BsRequest#action_details
```

Comment:
I don't understand this comment. What do you mean?

Reply:
We droped this comment, it makes no sense anymore.